internal/cli/cli.go

```diff
@@ -207,18 +207,23 @@ func (c *cli) prepareTenant(ctx context.Context) (Tenant, error) {
 		return Tenant{}, err
 	}
 
-	if t.AccessToken == "" || (scopesChanged(t) && t.authenticatedWithDeviceCodeFlow()) {
-		return RunLoginAsUser(ctx, c, true)
+	if scopesChanged(t) && t.authenticatedWithDeviceCodeFlow() {
+		c.renderer.Warnf("Required scopes have changed. Please sign in to re-authorize the CLI.")
```

Comment:
```suggestion
		c.renderer.Warnf("Required scopes have changed. Please log in to re-authorize the CLI.")
```
If we're using `login` as the login command, maybe use "log in" instead of "sign in" here?